Makefile

```diff
@@ -214,7 +214,31 @@ jaeger-ui/packages/jaeger-ui/build/index.html:
 
 .PHONY: rebuild-ui
 rebuild-ui:
-	cd jaeger-ui && yarn install --frozen-lockfile && cd packages/jaeger-ui && yarn build
+	cd jaeger-ui
+	
+	if git describe --exact-match --tags $(git rev-parse HEAD) >/dev/null 2>&1; then
```

Comment:
Makefiles don't work with such multi-line scripts, unless you escape them. It may be easier to move all logic into `scripts/rebuild-ui.sh`